Tests/DequeTests/DequeTests.swift

```diff
@@ -369,6 +369,23 @@ final class DequeTests: CollectionTestCase {
       }
     }
   }
+  
+  func test_equal() {
+    let deque: Deque<Int> = [1, 2, 3, 4]
+    let copy = deque
+    expectEqual(deque, copy)
+  }
+  
+  func test_not_equal() {
+    let left: Deque<Int> = [1, 2, 3, 4]
+    let right: Deque<Int> = [1, 2, 3, 4, 5]
+    expectNotEqual(left, right)
+  }
+  
+  func test_equal_elements() {
+    let deque: Deque<Int> = [1, 2, 3, 4]
+    expectEqualElements(deque, [1, 2, 3, 4])
+  }
```

Comment:
Similarly, the `test_CollectionConformance` test is already systematically checking that deques (of various shapes) do always produce the expected contents. (It also checks all possible methods of iterating through the deque, not just the one that `elementsEqual` uses.) This test can therefore be safely omitted.

```suggestion
```